monitoring/ceph-mixin/tests_dashboards/features/environment.py

```diff
@@ -132,4 +133,4 @@ def step_impl(context, query):
     for row in context.table:
         metric = row['metrics']
         value = row['values']
-        global_context.promql_expr_test.add_exp_samples(metric, float(value))
+        global_context.promql_expr_test.add_exp_samples(metric, float(value))
```

Comment:
@frittentheke , here's the environment.py file am talking about. This needs to have a blank line at the end.

Reply:
Uhh how embarrasing ... I simply had a typo in my `find` command ... just looked at the shell history :-(
Fixed by adding a newline now.